modules/ROOT/pages/_partials/conf-dev-env-cordova.adoc

```diff
@@ -15,5 +15,5 @@ const aerogearConfig = {
     // Replace with your own configuration
 };
 
-init(aerogearConfig);
```

Comment:
IMHO this is just showing that our flow and documentation snippets are really confusing. It's almost impossible to find that info (and I know about it).  Documentation for this part should be included in each individual SDK so I do not need to jump and find how to configure things. Adding as placeholder for myself to improve stuff afetr.

Reply:
@wtrocki  yes, you're right ;)
we should revisit this change

Reply:
Agree this needs to be called out in each as our documentation flow doesn't ever get to this point see https://github.com/aerogear/mobile-docs/issues/654